frontend/src/components/uam/components/group-create/group-create.tsx

```diff
@@ -0,0 +1,86 @@
+import { AppRoute, PaginationDefaultValue } from 'common/enums/enums';
+import { FC, GroupsCreateRequestDto } from 'common/types/types';
+import { Button, Input } from 'components/common/common';
+import {
+  useAppDispatch,
+  useAppForm,
+  useAppSelector,
+  useEffect,
+  useNavigate,
+} from 'hooks/hooks';
+import { uamActions } from 'store/actions';
+import { groupCreate } from 'validation-schemas/validation-schemas';
+
+import { DEFAULT_CREATE_GROUP_PAYLOAD } from './common/default-create-group-payload';
+import { GroupCreationFieldsName } from './common/enums/enums';
+import { PermissionsTable } from './components/permissions-table/permissions-table';
+import { UsersTable } from './components/users-table/users-table';
+import styles from './styles.module.scss';
+
+const UAMGroupsCreate: FC = () => {
+  const dispatch = useAppDispatch();
+  const { users, permissions } = useAppSelector((state) => state.uam);
+  const navigate = useNavigate();
+  const { register, control, handleSubmit, errors } =
+    useAppForm<GroupsCreateRequestDto>({
+      defaultValues: DEFAULT_CREATE_GROUP_PAYLOAD,
+      validationSchema: groupCreate,
+    });
+
+  const handleCancelClick = (): void => {
+    navigate(AppRoute.UAM);
+  };
+
+  const onSubmit = (data: GroupsCreateRequestDto): void => {
+    dispatch(uamActions.createGroup(data));
+  };
+
+  useEffect(() => {
+    dispatch(
+      uamActions.getUsers({
+        page: PaginationDefaultValue.DEFAULT_PAGE,
+        count: PaginationDefaultValue.DEFAULT_COUNT,
+      }),
+    );
+    dispatch(uamActions.getPermissions());
+  }, []);
+
+  const useFormData = {
+    register,
+    control,
+    errors,
+  };
+
+  return (
+    <div className={styles.groupCreationMain}>
+      <form onSubmit={handleSubmit(onSubmit)} className={styles.groupForm}>
+        <div className={styles.groupFormHeaderWrapper}>
+          <h2 className={styles.groupFormHeading}>Create group</h2>
+        </div>
+        <Input
```

Comment:
```suggestion
      <form onSubmit={handleSubmit(onSubmit)} className={styles.groupForm}>
        <div className={styles.groupFormHeaderWrapper}>
          <h2 className={styles.groupFormHeading}>Create group</h2>
        </div>
        <Input
      <form>
```
`useForm` should be only for group name